developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -16,6 +16,8 @@ Foreman will run with the following requirements (aside from rubygem dependencie
 * PostgreSQL 13
 
 It’s recommended to install https://github.com/rbenv/rbenv[rbenv] or https://github.com/rvm/rvm[rvm] to manage ruby, and https://github.com/nvm-sh/nvm[nvm] to manage node and npm on your system.
+* For Fedora users, you can use this https://developer.fedoraproject.org/start/sw/web-app/rails.html[guide] for installing: rbenv, Ruby, Bundler & Rails.
```

Comment:
no need to install Rails globally,
it will be installed once the developer runs `bundle install` from Foreman dir.

Reply:
OK I'll change and mention it in this section 👍🏼